content/docs/release-notes.md

```diff
@@ -2,7 +2,60 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Safekeeper: Back up WAL to S3 for disaster recovery.
+
+* Safekeeper: Download WAL from S3 on demand.
+
+* Propagate postgres authentication errors to the user.
+
+* Rename `contrib/zenith` to `contrib/neon`.
+* Rename custom GUCs:
+    * `zenith.page_server_connstring` -> `neon.pageserver_connstring`
+    * `zenith.zenith_tenant` -> `neon.tenant_id`
+    * `zenith.zenith_timeline` -> `neon.timeline_id`
+    * `zenith.max_cluster_size` -> `neon.max_cluster_size`
+
+* Rename `zenith_admin` role to `cloud_admin`
+* Rename `wal_acceptors` GUC to `safekeepers`
```

Comment:
```suggestion
* Postgres: Rename `wal_acceptors` GUC to `safekeepers`
```

Reply:
I think that GUC is mostly used in the Postgres dev community :) Wide audience knows it as settings or parameters https://www.postgresql.org/docs/current/runtime-config.html

IIRC, GUC isn't mentioned in the official docs